corrector.rb

```diff
@@ -0,0 +1,5 @@
+class Corrector
```

Comment:
- [ ] Nameable interface is missing. Kindly ensure to create a `Nameable` class with a method 'correct_name' rasing an error, like the following:
```ruby
class Nameable
    def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
end
```